commit 716a9c8561d9c50ec454f4fbd39a265892feda2c
Author: Moritz Muehlenhoff <jmm@debian.org>
Date:   Sat Mar 14 00:43:21 2009 +0100

    USB: Updated unusual-devs entry for USB mass storage on Nokia 6233
    
    Current firmware revision 5.60 still behaves the same,
    so update the quirk up a (non-existing) 99.99 revision.
    
    http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=493415
    
    Signed-off-by: Moritz Muehlenhoff <jmm@debian.org>
    Tested-by: Jan Heitkoetter <devnull@heitkoetter.net>
    Cc: stable <stable@kernel.org>
    Signed-off-by: Phil Dibowitz <phil@ipom.com>

diff -urpN a/drivers/usb/storage/unusual_devs.h b/drivers/usb/storage/unusual_devs.h
--- a/drivers/usb/storage/unusual_devs.h	2009-03-22 22:54:59.000000000 -0600
+++ b/drivers/usb/storage/unusual_devs.h	2009-03-22 22:57:59.000000000 -0600
@@ -236,7 +236,7 @@ UNUSUAL_DEV(  0x0421, 0x047c, 0x0370, 0x
 		US_FL_MAX_SECTORS_64 ),
 
 /* Reported by Manuel Osdoba <manuel.osdoba@tu-ilmenau.de> */
-UNUSUAL_DEV( 0x0421, 0x0492, 0x0452, 0x0452,
+UNUSUAL_DEV( 0x0421, 0x0492, 0x0452, 0x9999,
 		"Nokia",
 		"Nokia 6233",
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
